src/main/cliapp/src/service/useControlledVocabularyService.js

```diff
@@ -28,7 +28,7 @@ export function useControlledVocabularyService(termType) {
 
 	useQuery(['terms', termType],
 		() => {
-			return searchService.find("vocabularyterm", 15, 0, {"vocabulary.name" : termType } )
+			return searchService.find("vocabularyterm", 16, 0, {"vocabulary.name" : termType } )
```

Comment:
@markquintontulloch I am curious about this change?

Reply:
Well this is for ALL vocabs... why not 20 or 30?

Reply:
because maybe that's too many? For other vocabularies with lots of terms we have used autosuggest instead.  If we want to increase further for future vocabularies then we can do that at the time.

Reply:
if Gene notes needs 17 and Allele notes need 16. In other words changing it for Allele's is changing it for ALL, and I would guess he doesn't know that nuance?

Reply:
How about I bump it to 20 then?  If a particular number of terms is acceptable for the dropdown for one vocabulary then I'm sure it would be acceptable for all.  And if it's not then we are not forced to use the dropdown and would use the autocomplete anyway.

Reply:
In other words why not just set it to 30, and if there is a pulldown somewhere that has to many, then a new story to switch it out for an autocomplete.

Reply:
Doesn't the search service also get used for autosuggest though?  I don't think we want to return 30 terms in the autosuggest.  That's why I went with 16 - the smallest possible change in functionality to accommodate the current use case.

Reply:
I guess my point is that so long as Chris knows this is effecting ALL, then great lets make the change. Otherwise can we choose an acceptable amount that will work for ALL and then not have to change this again.

Reply:
I've asked Chris on the ticket - will wait for his response